### PR TITLE
Dynamic tables

### PR DESCRIPTION
**Pull request by mbevko**
It's not dynamic yet, but I had to scrap one of my branches and I wanted to have the latest table ui added to the main. I'm working on adding multiple table functionality now, that will be my next pull request hopefully later today.

### CODE REVIEW

**Comment by mbevko** [comment]
I have one example of a dynamic table. I only linked it to the transfer page, so make sure to test it you click transfer from the drop down menu. I'm also looking for a better way to link the data to the page. Write now it's a click function but I'm working on getting it to check the url and  change the table accordingly.  

Also I removed <Table /> from appshell and put it on the header page so it can track the state.

**Comment by tevko** [comment]
You want to take this out of the header, and import it into each component individually

**Comment by tevko** [comment]
So for example, [src](https://github.com/tevko/ads-osn-portal/tree/main/src)/[js](https://github.com/tevko/ads-osn-portal/tree/main/src/js)/[components](https://github.com/tevko/ads-osn-portal/tree/main/src/js/components)/Invoices.jsx should have a line `import Tables from "..."` and then the `rows` and `columns` properties should accept data from the fetch call

**Comment by tevko** [comment]
```jsx
import React from "react";
import useFetch from "../hooks/useFetch";
import Tables from "./tables"

export default function Invoices() {
  const { data, error, loading } = useFetch(`${window.API_BASE_URL}/invoices`);
  if (loading) return <p>Loading...</p>;
  if (data) return <Tables rows={placeholderData} columns={placehoderData}
  if (error) return <p>Error: {error.message}</p>;
  return <p>{data.message}</p>;
}
```

File: src/js/components/PurchaseOrders.jsx [hunk 1]
```diff
@@ -1,11 +1,22 @@
...
+import Tables from "./Tables";
+import columnsOne from "./data/Column";
+import rowsOne from "./data/Rows";
 
 export default function PurchaseOrders() {
   const { data, error, loading } = useFetch(
     `${window.API_BASE_URL}/purchase-orders`
   );
   if (loading) return <p>Loading...</p>;
+  if (data)
```
Comment (tevko) [inline src/js/components/PurchaseOrders.jsx:12]: Change to `if (true)` for now

**Review by tevko — CHANGES_REQUESTED** [review]

**Pull request merged**